docker-compose.yml

```diff
@@ -11,20 +11,25 @@ app:
     - DATABASE_URL=postgres://tock_user@db/tock
     - RUNNING_IN_DOCKER=yup
     - DJANGO_SETTINGS_MODULE=tock.settings.dev
-  command: "python manage.py runserver 0.0.0.0:${EXTERNAL_PORT}"
+  command: "python manage.py runserver 0.0.0.0:1234"
 db:
   image: postgres:9.4
   environment:
     - POSTGRES_DB=tock
     - POSTGRES_USER=tock_user
 web:
-  image: nginx
+  image: nginx:1.11
```

Comment:
If the version isn't specified I assume it pulls the latest image, and I take it this change is just to account for the fact that we always know what version we're running (much like any other project dependencies)?